docs/configuration/registration/json-dump.rst

```diff
@@ -0,0 +1,77 @@
+.. _configuration_registration_json_dump:
+
+=========
+JSON Dump
+=========
+
+The JSON Dump registration plugin can be used to send submitted form data as a
+`JSON object`_ to a configured service.
+
+What does the Open Forms administrator need?
+============================================
+Service options:
+
+* API root URL
+* API credentials (optional)
+* Client certificate (optional)
+* Server certificate (optional)
+
+Plugin options:
+
+* Which form variables to include in the data
+* API endpoint (optional)
+
+
+Configuration
+=============
+To configure the JSON Dump plugin, follow these steps:
+
+#. In Open Forms, navigate to: **Configuration** > **Services**
```

Comment:
We can decide if we want a period in the end of each one or not (I see some use it and some not, so let's be consistent), quick check in the other docs shows that it's not so consistent though.